docs/changelog.rst

```diff
@@ -3,6 +3,8 @@
 =========
 Changelog
 =========
+* :feature:`520`: Remove ``no_positional`` decorator in favor of native
```

Comment:
This was only ever internal functionality so it doesn't need a changelog entry

Reply:
Agreed - it's not required, though it still may prove useful for a maintainer to reference to know when that change occurred (and with which version). I don't intend for it to have user-facing impact (such as driving a major version bump). Do you reserve the changelog for changes that only involve end-user intervention?

Reply:
The intended audience for changelogs is usually end-users (bug fix for x, new feature for y, etc.). I don't bother cataloguing internal-only changes there since that tends to (in my anecdotal experience) confuse users.

Reply:
Very well. I'll remove it.

Reply:
Done in 59ffbb3.